assets/js/modules/pagespeed-insights/settings/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { sanitizeHTML } from '../../../util';
```

Comment:
Storybook failed to build when I used `assets/js/util` here, although it worked for the normal build. We'll leave this for #1135